lib/actions/getSubOrderByReference.js

```diff
@@ -12,7 +12,7 @@ function processAction(message, config) {
     let { orderReference } = message.body.currentMessage;
     let options = Object.assign({}, authenticationHeaders, {
       json: true,
-      url: `${authenticationHeaders.url}sub-orders?filter[reference]=${orderReference}`
+      url: `${authenticationHeaders.url}sub-orders?include=line-items&filter[reference]=${orderReference}`
```

Comment:
I think it would be good to have the VMS include line items in sub orders by default.